<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0" tests="2" failures="1">
  <testsuite name="Root Suite" timestamp="2020-05-30T16:53:19" tests="0" file="cypress/integration/authentification/Create-Delete-Account.spec.js" failures="0" time="0">
  </testsuite>
  <testsuite name="Authentification" timestamp="2020-05-30T16:53:19" tests="0" failures="0" time="0">
  </testsuite>
  <testsuite name="Account" timestamp="2020-05-30T16:53:19" tests="2" failures="1" time="0">
    <testcase name="Authentification Account &quot;before all&quot; hook for &quot;Create An account&quot;" time="0" classname="&quot;before all&quot; hook for &quot;Create An account&quot;">
      <failure message="`cy.visit()` failed trying to load:

http://localhost:3000/login

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  &gt; Error: connect ECONNREFUSED 127.0.0.1:3000

Common situations why this would fail:
  - you don&apos;t have internet access
  - you forgot to run / boot your web server
  - your web server isn&apos;t accessible
  - you have weird network configuration settings on your computer

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Account`" type="CypressError"><![CDATA[CypressError: `cy.visit()` failed trying to load:

http://localhost:3000/login

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  > Error: connect ECONNREFUSED 127.0.0.1:3000

Common situations why this would fail:
  - you don't have internet access
  - you forgot to run / boot your web server
  - your web server isn't accessible
  - you have weird network configuration settings on your computer

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Account`
    at http://localhost:45713/__cypress/runner/cypress_runner.js:145388:23
    at visitFailedByErr (http://localhost:45713/__cypress/runner/cypress_runner.js:144763:12)
    at http://localhost:45713/__cypress/runner/cypress_runner.js:145387:11
    at tryCatcher (http://localhost:45713/__cypress/runner/cypress_runner.js:10384:23)
    at Promise._settlePromiseFromHandler (http://localhost:45713/__cypress/runner/cypress_runner.js:8319:31)
    at Promise._settlePromise (http://localhost:45713/__cypress/runner/cypress_runner.js:8376:18)
    at Promise._settlePromise0 (http://localhost:45713/__cypress/runner/cypress_runner.js:8421:10)
    at Promise._settlePromises (http://localhost:45713/__cypress/runner/cypress_runner.js:8497:18)
    at _drainQueueStep (http://localhost:45713/__cypress/runner/cypress_runner.js:5091:12)
    at _drainQueue (http://localhost:45713/__cypress/runner/cypress_runner.js:5084:9)
    at Async.../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:45713/__cypress/runner/cypress_runner.js:5100:5)
    at Async.drainQueues (http://localhost:45713/__cypress/runner/cypress_runner.js:4970:14)
From Your Spec Code:
    at Function.load (http://localhost:45713/__cypress/tests?p=cypress/integration/authentification/Create-Delete-Account.spec.js:130:10)
    at Context.eval (http://localhost:45713/__cypress/tests?p=cypress/integration/authentification/Create-Delete-Account.spec.js:197:29)

From Node.js Internals:
  Error: connect ECONNREFUSED 127.0.0.1:3000
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1128:14)
  ]]></failure>
    </testcase>
  </testsuite>
</testsuites>